include/rocksdb/options.h

```diff
@@ -392,6 +393,8 @@ struct DBOptions {
   // Default: Env::Default()
   Env* env = Env::Default();
 
+  std::shared_ptr<FileSystem> file_system = nullptr;
```

Comment:
`=nullptr` isn't necessary, IIRC

Reply:
Right, because its a shared_ptr and not raw.

Reply:
Although all other shared_ptr options in options.h initializes using ```= nullptr```, so maybe just leave it as is for consistency.